src/routes/photos.py

```diff
@@ -25,19 +25,41 @@
 security = HTTPBearer()
 
 
-@router.get("/{photo_id}", response_model=PhotoResponse)
-async def get_photo(photo_id: int,
-                    db: Session = Depends(get_db)):
-    photo = await repository_photos.get_photo_by_photo_id(photo_id, db)
-    if not photo:
-        raise HTTPException(status_code=404, detail="Photo not found")
-    return PhotoResponse(
-        id=photo.id,
-        url=photo.url,
-        description=photo.description,
-        created_by=photo.created_by,
-        created_at=photo.created_at,
-    )
+@router.get("/", response_model=Union[List[PhotoResponse], PhotoResponse])
```

Comment:
The endpoint should return all data with pagination without query parameters and parameters works as filter for returned data to get something specific as we discussed.